sale_credit_point/models/sale_order.py

```diff
@@ -34,3 +34,13 @@ def action_confirm(self):
             sale.partner_id.credit_point_decrease(
                 sale.amount_total, comment=self.credit_point_decrease_msg)
         return super().action_confirm()
+
+    @api.multi
+    def action_cancel(self):
+        for sale in self:
+            if sale.state == 'sale':
+                sale.partner_id.credit_point_increase(
+                    sale.amount_total, "Sale Order canceled")
+                sale.partner_id.update_history(
+                    sale.amount_total, "increase", "Sale Order canceled")
```

Comment:
No translations?

Reply:
Is it normal that you have 2 calls which seems pretty similar (disclaimer: I didn't looked at the methods)